internal/discord/command-handlers.go

```diff
@@ -0,0 +1,396 @@
+package discord
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+// Set up vars for the DefaultMemberPermissions field in each command definition
+var (
+	adminPermission int64 = discordgo.PermissionAdministrator
+)
+
+// AddCommands registers the slash commands with Discord
+func (d *Discord) AddCommands(s *discordgo.Session, event *discordgo.Ready) {
+	fmt.Printf("Initializing Discord...\n")
+
+	for _, discordGuild := range event.Guilds {
+
+		// Adding commands to a list to prepare in bulk
+		var commands []*discordgo.ApplicationCommand
+		commands = append(commands,
+			KickCommand,
+			MuteCommand,
+			UnmuteCommand,
+			BanCommand,
+			UnbanCommand,
+			RemoveNicknameCommand,
+			SetNicknameCommand,
+			SlowmodeCommand,
+			SlowmodeOffCommand,
+			PurgeCommand,
+			ExileCommand,
+			UnexileCommand,
+			SetModLogCommand,
+		)
+
+		fmt.Printf("Adding commands...\n")
+		commandList, err := s.ApplicationCommandBulkOverwrite(event.User.ID, discordGuild.ID, commands)
+		fmt.Printf("List of successfully created commands:\n")
+		for _, command := range commandList {
+			fmt.Printf("\t%v\n", command.Name)
+		}
+		if err != nil {
+			fmt.Printf("Could not add some commands: %v \n", err)
+		}
+	}
+}
+
+// CheckUserInGuild checks if the user is in the specified server.
+func (d *Discord) CheckUserInGuild(gid string, user string) error {
+	_, err := d.Session.GuildMember(gid, user)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (d *Discord) CommandLogger(i *discordgo.Interaction) error {
```

Comment:
same thing here, let's add a doc string. let's also change this func name to `LogCommand`